doc/explanation/storage.md

```diff
@@ -11,12 +11,12 @@ If required, you can create additional storage pools later (see {ref}`storage-cr
 Each storage pool uses a storage driver.
 The following storage drivers are supported:
 
-- {ref}`storage-dir`
-- {ref}`storage-btrfs`
-- {ref}`storage-lvm`
-- {ref}`storage-zfs`
-- {ref}`storage-ceph`
-- {ref}`storage-cephfs`
+- [Directory - `dir`](storage-dir)
+- [Btrfs - `btrfs`](storage-btrfs)
+- [LVM - `lvm`](storage-lvm)
+- [ZFS - `zfs`](storage-zfs)
+- [Ceph RBD - `cephfs`](storage-ceph)
+- [CephFS - `cephfs`](storage-cephfs)
```

Comment:
Doesn't really look like a spelling fix here, what's this doing?

Reply:
Sphinx gets rid of the `<code>` tags inside links when generating the text automatically, and then we'll get spelling errors for btrfs, lvm etc.
Therefore, if there is a code part inside the link text, we need to override the link text to make sure it is preserved.